blocks/api/raw-handling/is-inline-content.js

```diff
@@ -12,10 +61,3 @@ export default function( HTML, tagName ) {
 
 	return ! nodes.some( isDoubleBR ) && deepCheck( nodes, tagName );
```

Comment:
What does double BR have to do with HTML being inline?

Reply:
This is not changed by this PR. But when plain text is pasted, and there's a double line break, we should not consider it inline.

Reply:
```
First paragraph.

Second paragraph.
```